addons/mail/static/src/components/thread_view/thread_view.js

```diff
@@ -202,6 +202,10 @@ Object.assign(ThreadView, {
             type: String,
             optional: true,
         },
+        getScrollableElement: {
+            type: Function,
+            optional: true,
+        },
```

Comment:
Same remark.

If don't want to duplicate comment, can do it once in `ThreadView` and refer to the props of `ThreadView` in `MessageList`.